game-core/src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -102,6 +107,35 @@ public static GameData loadGame(final InputStream is) throws IOException {
     }
   }
 
+  private static void promptToLoadNewerSaveGame(final Version version) throws IOException {
+    // we can still load it because our engine is compatible, however this save was made by a
```

Comment:
This comment does not seem to belong in this method. The comment is describing why we can load a newer save game, but the method is all about prompting and warning the user.

Reply:
I moved the comment to the caller.